test/slow/nvm use/ Running "nvm use" displays .nvmrc version in output messages

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```

Comment:
Also, can you add `set -e` - either here, or after the line that sources `nvm.sh`, if the tests fail when it's here?